components/brave_news/browser/urls.cc

```diff
@@ -7,12 +7,30 @@
 
 #include <string>
 
+#include "base/strings/strcat.h"
 #include "brave/brave_domains/service_domains.h"
 
 namespace brave_news {
 
+namespace {
+const std::string kBraveNewsHostnamePrefix = "brave-today-cdn";
+const std::string kPCDNHostnamePrefix = "pcdn";
+}  // namespace
+
 std::string GetHostname() {
-  return brave_domains::GetServicesDomain("brave-today-cdn");
+  return brave_domains::GetServicesDomain(kBraveNewsHostnamePrefix);
+}
+
+std::string GetMatchingPCDNHostname() {
```

Comment:
Should this just be `GetPCDNHostname`? It isn't really matching anything.

```suggestion
std::string GetPCDNHostname() {
```

Reply:
I read that as "matching the suffix of the other one" (i.e. `.bravesoftware.com` v. `.brave.com`).

Reply:
Yeah I know it's confusing because I couldn't think of a better name. The reason I didn't use `GetPCDNHostname` is because I thought someone would probably assume that it was using the standard `brave_domains::GetServicesDomain("pcdn")` but it isn't because it wants to match what is used for `brave_domains::GetServicesDomain("brave-today-cdn")`. Happy to take suggestions...

Reply:
Added some unittests for this